BUILD

```diff
@@ -0,0 +1,126 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_binary(
```

Comment:
Consider moving these build targets into the directory that they reference.  For example, this rule should probably go in a BUILD file under compiler/